src/WebJobs.Script.SiteExtension/WebJobs.Script.SiteExtension.csproj

```diff
@@ -17,9 +17,8 @@
   <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
     <SiteExtensionName>FunctionsInProc8</SiteExtensionName>
   </PropertyGroup>
-
-  <!-- Only .NET6 builds include workers. -->
-  <Import Project="$(WorkersProps)" Condition="'$(TargetFramework)' == 'net6.0'" />
+  
+  <Import Project="$(WorkersProps)" />
```

Comment:
Do they need all workers or only a few? Should we consider including only the subset they need?

Reply:
yeah, seems that they do